### PR TITLE
Change random bits and pieces

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

A bunch of changes that are too tiny that they don't deserve their own commit/PR.

This is much needed imo.

# Result

Too many stuff to show individually, but Iif you have any question, just let me know and I'll show the improvement. Changes should be very clear though.

# Testing

- [x] I have thoroughly tested my changes.

Changes are all small and were easy to test.

### CODE REVIEW

File: app/routes/events/components/EventDetail/index.tsx [hunk 1]
```diff
@@ -354,246 +354,236 @@ export default class EventDetail extends Component<Props, State> {
     ];
 
     return (
-      <div>
```
Comment (ivarnakken) [inline app/routes/events/components/EventDetail/index.tsx:357]: I just removed this useless `<div>`.

File: app/components/Form/Field.css [hunk 2]
```diff
@@ -16,7 +16,7 @@
 }
 
 .fieldError {
-  height: 35px;
+  min-height: 35px;
```
Comment (ivarnakken) [inline app/components/Form/Field.css:19]: Caused overflow issues:

image

File: app/components/Reactions/ReactionPickerFooter.css [hunk 3]
```diff
@@ -8,4 +8,5 @@
   padding: 15px;
   background-color: var(--lego-card-color);
   border-top: 1px solid var(--border-gray);
+  border-radius: 0 0 var(--border-radius-lg) var(--border-radius-lg);
```
Comment (ivarnakken) [inline app/components/Reactions/ReactionPickerFooter.css:11]: There were overflow issues on the footer:

image

File: app/routes/companyInterest/components/CompanyInterestList.tsx [hunk 4]
```diff
@@ -155,7 +155,12 @@ class CompanyInterestList extends Component<Props, State> {
     return (
       <Content>
         <ListNavigation title="Bedriftsinteresser" />
-        <Flex className={styles.section}>
+        <Flex
+          wrap
```
Comment (ivarnakken) [inline app/routes/companyInterest/components/CompanyInterestList.tsx:159]: Needed a `wrap`:

image

**Review by LudvigHz — APPROVED** [review]
LGTM
Nice :100:

**Pull request merged**